go/pkg/client/cas.go

```diff
@@ -36,6 +36,14 @@ const DefaultCompressedBytestreamThreshold = -1
 // DefaultMaxHeaderSize is the defaut maximum gRPC header size.
 const DefaultMaxHeaderSize = 8 * 1024
 
+// MovedBytesMetadata represents the bytes moved in CAS related requests.
+type MovedBytesMetadata struct {
+	Requested    int64
+	LogicalMoved int64
+	RealMoved    int64
+	Cached       int64
```

Comment:
Do we really need Cached? We have two scenarios:
1. Unified uploader
2. Compression

When logicalmoved == realmoved, no compression, actually downloaded realmoved bytes
When logicalmoved > realmoved && realmoved > 0, compression, actually downloaded realmoved bytes
When logicalmoved > realmoved && realmoved == 0, didn't download anything (used unified uploader)

Reply:
We discussed this offline, so lets keep it there :)